### PR TITLE
Ud answers

### PR DESCRIPTION
**Pull request by nguyenvanson332000**
## Checklist
- [ ] This is bug/feature (delete incorrect one)
- [ ] Title including ticket number and title

## What this PR do?
Explain clearly and briefly what this PR does. Make a meaningful sentence instead of "this PR solve ticket #xxx"

## Context/Background
More information on the issue that you want to provide, related issues and PRs you have to write information to ask orally or in informal place too.

## Screenshots / Demo & Instruction for testing (if applicable)
image
image

## References
- Blog posts / web pages relevant to this issue (which you get your solution / idea from and it may help others to understand the issue quickly)
- Do we need update our knowledge base / project wiki?

### CODE REVIEW

File: app/controllers/admin/answers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+class Admin::AnswersController < ApplicationController
+  before_action :load_exam, :load_question, only: %i(new create edit update destroy)
```
Comment (rikai-hai) [inline app/controllers/admin/answers_controller.rb]: dung except

File: app/controllers/admin/answers_controller.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    redirect_to admin_exam_path(@exam)
+  end
+
+  private
+
+  def answer_params
+    params.require(:answer).permit :content, :is_correct_answer
+  end
+
+  def load_exam
```
Comment (rikai-hai) [inline app/controllers/admin/answers_controller.rb]: gộp tất cả lại  thành 1, load_resources

Comment (nguyenvanson332000) [reply app/controllers/admin/answers_controller.rb]: dạ

File: app/controllers/admin/answers_controller.rb (hunk 1 above)
Comment (nguyenvanson332000) [reply app/controllers/admin/answers_controller.rb]: dạ

File: app/controllers/admin/answers_controller.rb (hunk 2 above)
Comment (nguyenvanson332000) [reply app/controllers/admin/answers_controller.rb]: dạ,anh Hải  em gộp cái load_exam và load_question thành load_resources, còn load_answer em không gộp vì load_answer không phù hợp cho create và new

**Pull request merged**